Main.java

```diff
@@ -1,15 +1,30 @@
+import java.util.Scanner;
+
 /**
- *
- * @author 
+ *This program converts degrees celcius to degrees fahrenheit
+ * @author Tolga Nurtekin
  */
 public class Main {
 
   /**
    * The method that is executed when you hit the run button.
    * @param args the command line arguments
-   */
-  public static void main(String[] args) {
-    // TODO code application logic here
-    
+   */
+   public static void main(String[] args) {
+   //create a Scanner for user input
+   Scanner input = new Scanner(System.in);
+
+   // declare a variable that will store a name
+   int Celcius;
+  
```

Comment:
Remember that variable names should always begin with a lowercase letter. Celsius is also more of a unit of measure and not a great name to describe what you are storing. degreeC or tempInC would probably be better to use.